messages/run.csx

```diff
@@ -0,0 +1,80 @@
+#r "Newtonsoft.Json"
+#load "EchoDialog.csx"
+
+using System;
+using System.Net;
+using System.Threading;
+using Newtonsoft.Json;
+
+using Microsoft.Bot.Builder.Azure;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Connector;
+
+// Bot Storage: Register the optional private state storage for your bot. 
+
+// (Default) For Azure Table Storage, set the following environment variables in your bot app:
+// -UseTableStorageForConversationState set to 'true'
+// -AzureWebJobsStorage set to your table connection string
+
+// For CosmosDb, set the following environment variables in your bot app:
+// -UseCosmosDbForConversationState set to 'true'
+// -CosmosDbEndpoint set to your cosmos db endpoint
+// -CosmosDbKey set to your cosmos db key

```

Comment:
No need for these comments, this bot should never require private state storage (or for that matter, hold any state at all between requests)

Reply:
removed